service/internal/server/server.go

```diff
@@ -263,6 +269,14 @@ func newGrpcServer(c Config, a *auth.Authentication) (*grpc.Server, error) {
 		i...,
 	))
 
+	// Chain relaip interceptor
+	trustedPeers := []netip.Prefix{} // TODO: add this as a config option?
```

Comment:
@pbacon-blaber Can we track this enhancement in an issue.

Reply:
https://github.com/opentdf/platform/issues/957